### ISSUE TITLE
Role-specific defaults should be addressed as reflectable in the idl or in normative prose

### ISSUE THREAD

**Issue by joanmarie** [w3c/aria#1585]
(See issue #1584 for including more explicit language on user agents doing the reflection as shown in the examples.)

The text which is currently in the spec does not take role-specific default values (e.g. `aria-orientation` on a `scrollbar` versus `aria-orientation` on a `progressbar`).

**Comment by cyns** [comment]
Relevant section of the HTML5 spec https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflecting-content-attributes-in-idl-attributes

**Comment by cookiecrook** [comment]
@rniwa mentioned on Tuesday that several things are now allowed in IDL that weren't a few years ago... There was a previous issue #1058 where reflected string attributes could not be nullable, but that may have changed.  Ryosuke, could you summarize your comments about what could/should change wrt the ARIA IDL? Thank you.
